### PR TITLE
Home instance endpoints

### PR DESCRIPTION
**Pull request by BaetensJan**
Added:

v2/home/cats
v2/home/nodes

### CODE REVIEW

File: model/src/home_instance/nodes.rs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+}
+impl Endpoint for Node {
+    const AUTHENTICATED: bool = false;
+    const LOCALE: bool = false;
+    const URL: &'static str = "v2/home/nodes";
+    const VERSION: &'static str = "2023-08-14T00:00:00.000Z";
+}
+
+impl BulkEndpoint for Node {
+    const ALL: bool = false;
```
Comment (greaka) [inline model/src/home_instance/nodes.rs:24]: This endpoint supports `ALL`

File: model/src/home_instance/cats.rs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+}
+impl Endpoint for Cat {
+    const AUTHENTICATED: bool = false;
+    const LOCALE: bool = false;
+    const URL: &'static str = "v2/home/cats";
+    const VERSION: &'static str = "2023-08-14T00:00:00.000Z";
+}
+
+impl BulkEndpoint for Cat {
+    const ALL: bool = false;
```
Comment (greaka) [inline model/src/home_instance/cats.rs:25]: This endpoint supports `ALL`

**Review by greaka — CHANGES_REQUESTED** [review]
Great stuff, ty!
Just a small oversight in `BulkEndpoint` to fix and it's good to go :slightly_smiling_face:

**Review by greaka — APPROVED** [review]

**Comment by BaetensJan** [comment]
Small oversight indeed. I've changed the bool value.

**Pull request merged**